playbooks/roles/elasticsearch/tasks/elasticsearch_post_install.yml

```diff
@@ -0,0 +1,75 @@
+---
+# Copyright 2015, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Allow elasticsearch to lock the memory
+  shell: ulimit -l unlimited
```

Comment:
This needs to be set for the elasticsearch user only, or all users. Currently this is done for root and takes no permanent effect, nor takes effect for the elasticsearch user.